inc/form_answer.class.php

```diff
@@ -643,19 +643,37 @@ public function saveAnswers($data) {
 
                // $answer_value may be still null if the field type is file and no file was uploaded
                if ($answer_value !== null) {
-                  // Update the answer to the question
-                  $questionId = $question->getID();
-                  $answer = new PluginFormcreatorAnswer();
-                  $answer->getFromDBByCrit([
-                    'AND' => [
-                      'plugin_formcreator_forms_answers_id' => $formanswers_id,
-                      'plugin_formcreator_questions_id'     => $questionId
-                    ]
-                  ]);
-                  $answer->update([
-                     'id'     => $answer->getID(),
-                     'answer' => $answer_value,
-                  ], 0);
+                  if (is_array($answer_value)) {
+                     foreach ($answer_value as $value) {
+                        // Update the answer to the question
+                        $questionId = $question->getID();
+                        $answer = new PluginFormcreatorAnswer();
+                        $answer->getFromDBByCrit([
+                           'AND' => [
+                             'plugin_formcreator_forms_answers_id' => $formanswers_id,
+                             'plugin_formcreator_questions_id'     => $questionId
+                           ]
+                        ]);
+                        $answer->update([
+                           'id'     => $answer->getID(),
+                           'answer' => $value,
+                        ], 0);
+                     }
+                  } else {
+                     // Update the answer to the question
+                     $questionId = $question->getID();
+                     $answer = new PluginFormcreatorAnswer();
+                     $answer->getFromDBByCrit([
+                       'AND' => [
+                         'plugin_formcreator_forms_answers_id' => $formanswers_id,
+                         'plugin_formcreator_questions_id'     => $questionId
+                       ]
+                     ]);
+                     $answer->update([
+                        'id'     => $answer->getID(),
+                        'answer' => $answer_value,
+                     ], 0);
+                  }
```

Comment:
If $answer_value is not an array, I think we can convert it onto an array then handle it with the foreach.

Check your PR is writazble by the maitnainers of the plugin, and I'll be able to push this refactor myself.

Reply:
Yeah, is writazble.